stocks2.lua

```diff
@@ -0,0 +1,544 @@
+-- a v50 script remake of the older stocks plugin
+-- chawkzero, Aug 2023
+
+local gui = require("gui")
+local widgets = require("gui.widgets")
+local utils = require("utils")
+
+local item_desc_width = 38
+local item_table = {}
```

Comment:
You shouldn't have to track this state yourself. It will be accessible as the choices in the FilteredList